distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -239,6 +239,7 @@ void execute(Terminal terminal, List<String> pluginIds, boolean isBatch, Environ
                 deleteOnFailure.add(extractedZip);
                 final PluginInfo pluginInfo = installPlugin(terminal, isBatch, extractedZip, env, deleteOnFailure);
                 terminal.println("-> Installed " + pluginInfo.getName());
+                terminal.println("-> Please restart Elasticsearch to activate " + pluginInfo.getName());
```

Comment:
Putting the notification here, inside the loop, will print a new line after every plugin that is installed, and double the output of the tool. In my opinion, we only need to issue the restart notification once, so it would be better to move this below the `catch` block to line 267, removing the plugin name from the output.

Reply:
Thanks, that is indeed a good suggestion!